content/projects/teachopencadd.md

```diff
@@ -0,0 +1,95 @@
+---
+title: TeachOpenCADD
+date: 2020-03-03
+publishdate: 2020-03-03
+nav: false
+weight: 1
+menu:
+    main:
+        parent: Education
+external_resources:
+- name: TeachOpenCADD for Jupyter
+  link: https://github.com/volkamerlab/TeachOpenCADD
+  icon: github
+  more: Jupyter notebooks on computer-aided drug design tasks using open resources
+- name: TeachOpenCADD for KNIME
+  link: https://hub.knime.com/volkamerlab/spaces/Public/latest/TeachOpenCADD/TeachOpenCADD
+  more: KNIME workflows on computer-aided drug design tasks using open resources
+people:
+- key: dominique.sydow
+- key: jaime.rodriguez
+- key: michele.wichmann
+- key: andrea-lilian.morger
+- key: maximilian.driller
+- key: andrea.volkamer
+collaborators:
+- name: Gregory Landrum
+  more: KNIME AG
+- name: Daria Goldmann
+  more: KNIME GmbH
```

Comment:
```suggestion
  more: KNIME
```

Reply:
Also would it be possible to link KNIME in the above two cases (https://www.knime.com)?